cli/install_ee/install_ee.go

```diff
@@ -0,0 +1,163 @@
+package install_ee
+
+import (
+	"bufio"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"regexp"
+	"strings"
+	"syscall"
+	"time"
+
+	"github.com/tarantool/tt/cli/config"
+	"github.com/tarantool/tt/cli/util"
+	"github.com/tarantool/tt/cli/version"
+	"golang.org/x/term"
+)
+
+const eeSource string = "https://download.tarantool.io/enterprise/"
+
+type creds struct {
```

Comment:
Suppose, this is credentials, not creds. These words have different meaning. loginCredentials or userCredentials will be more meaningful.

Reply:
Fixed